test/NuGet.Core.Tests/NuGet.Protocol.Tests/Providers/ReportAbuseResourceV3ProviderTests.cs

```diff
@@ -0,0 +1,143 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Threading.Tasks;
+using System.Threading;
+using Moq;
+using Newtonsoft.Json.Linq;
+using NuGet.Configuration;
+using NuGet.Protocol.Core.Types;
+using NuGet.Versioning;
+using Xunit;
+
+namespace NuGet.Protocol.Tests.Providers
+{
+    public class ReportAbuseResourceV3ProviderTests
+    {
+        private const string ResourceType = "ReportAbuseUriTemplate/3.0.0";
+
+        private readonly PackageSource _packageSource;
+        private readonly ReportAbuseResourceV3Provider _target;
+
+        public ReportAbuseResourceV3ProviderTests()
+        {
+            _packageSource = new PackageSource("https://unit.test");
+            _target = new ReportAbuseResourceV3Provider();
+        }
+
+        [Fact]
+        public async Task TryCreate_WhenResourceDoesNotExist_ReturnsFallbackUri()
+        {
+            var resourceProviders = new ResourceProvider[]
+            {
+                CreateServiceIndexResourceV3Provider(),
+                _target
+            };
+            var sourceRepository = new SourceRepository(_packageSource, resourceProviders);
+
+            var result = await _target.TryCreate(sourceRepository, CancellationToken.None);
```

Comment:
Use explicit type. This applies to cases where the variable type cannot be inferred when reading the code line.